R/formatter.r

```diff
@@ -265,6 +279,8 @@ trans_format <- function(trans, format = scientific_format()) {
 #'   \code{\link{format.POSIXct}}
 #' @export
 format_format <- function(...) {
+  force(list(...))
```

Comment:
Maybe make `force_all <- function(...) list(...)` just to make this and others slightly cleaner?